platform/mds/mds/src/main/java/org/motechproject/mds/jdo/SchemaGenerator.java

```diff
@@ -68,8 +69,13 @@ public void generateSchema() throws IOException {
 
         Set<String> classNames = classNames();
         if (!classNames.isEmpty()) {
-            SchemaAwareStoreManager storeManager = getStoreManager();
-            storeManager.createSchema(classNames, new Properties());
+            try {
+                SchemaAwareStoreManager storeManager = getStoreManager();
+                storeManager.createSchemaForClasses(classNames, new Properties());
+            }
+            catch (Exception|Error e) {
+                throw e;
+            }
```

Comment:
Is there any difference between this and just not catching at all (like it was before)?